mantis-publish-sample/buildDockerImage.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+# build the mantis-server-worker fat jar
+../gradlew clean build shadowDistZip
+unzip build/distributions/mantis-examples-mantis-publish-sample-shadow-0.1.0-SNAPSHOT.zip -d build/distributions 
+# build the Docker image that packages the mantis-server-worker along with a running mesos-slave
+docker build -t dev/mantispublish .
```

Comment:
Do we intend to publish this to dockerhub? should put as `netflixoss/mantispublish` if so.

Reply:
Yes plan to do that in the next iteration. Wanted to get the sample out first.